src/osd/OSDMap.cc

```diff
@@ -1642,11 +1642,11 @@ void OSDMap::get_out_of_subnet_osd_counts(CephContext *cct,
   for (int i = 0; i < max_osd; i++) {
     if (exists(i) && is_up(i)) {
       if (const auto& addrs = get_addrs(i).v; addrs.size() >= 2) {
-        auto v1_addr = addrs[0].ip_only_to_str();
+        auto v1_addr = addrs[0];
```

Comment:
well - we do not really need v[12]_addr anymore, it seems
(can use the addr[n] directly)